idaes_examples/notebooks/docs/unit_models/reactors/plug_flow_reactor_src.ipynb

```diff
@@ -540,9 +569,47 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
```

Comment:
It looks like the original version had the execution of the cells cleared, so we should clear the execution before committing here too.

Reply:
I think this is a good point in general, but I'm not sure what the cell execution state looks like for the other notebooks. Seeing that #44 will have an effect on this indirectly (as it requires the `_doc`, `_test`, etc generated notebooks to be checked in alongside the `_src` one), I'd rather save @agarciadiego some time and skip this for the time being.